dev/traffic_router/conf/log4j2.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+ -->
+<Configuration>
+	<Appenders>
+		<Console name="stdout" target="SYSTEM_OUT">
+			<PatternLayout pattern="%-5p %d{yyyy-MM-dd'T'HH:mm:ss.SSS} [%t] %c - %m%n"/>
+			<ThresholdFilter level="ALL"/>
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="org.apache.traffic_control.traffic_router" level="INFO" additivity="false">
+			<AppenderRef ref="stdout"/>
+		</Logger>
+		<Logger name="org.apache.traffic_control.traffic_router.core.access" level="INFO" additivity="false">
+			<AppenderRef ref="stdout"/>
+		</Logger>
+		<Root level="ALL" additivity="false">
```

Comment:
`level="ALL"` is very verbose for the root logger. The default log4j2 config uses `level="WARN"`:
https://github.com/apache/trafficcontrol/blob/8833d1c96b3bbdd973120a85e1b15e716afd4837/traffic_router/core/src/main/conf/log4j2.xml#L49

Reply:
Yeah, it's for development so I wanted it to be very verbose. Do you think the amount of logging is making it difficult for a developer to follow the logic flow? `"WARN"` isn't verbose enough IMO, but I definitely don't mind turning it down to `"DEBUG"`, since arguably if you really want a deep trace of the execution you'd just attach a debugger. Maybe `"INFO"` - I'm not sure precisely how different those are.

Reply:
We could try debug... There's a lot of messages printed not from TR, but from dependencies, IMO we should just not have those

Reply:
I'll turn it down and check the class names I'm seeing in the logs